virttest/env_process.py

```diff
@@ -415,6 +445,26 @@ def postprocess_vm(test, params, env, name):
     if not vm:
         return
 
+    # recover the changes done to kernel params in postprocess
```

Comment:
check for `start_vm` and then proceed, as we have set only if `start_vm` is set in pre-process.

Reply:
Thank you @sathnaga for reviewing and making code better.